test/Microsoft.Azure.Devices.Edge.Test.Common/config/ModuleConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Config
+{
+    using System.Collections.Generic;
+
+    public class ModuleConfiguration
+    {
+        public IReadOnlyDictionary<string, object> Deployment { get; }
+
```

Comment:
So, Deployment parts goes to the EdgeAgent twin? Interesting :)

Reply:
Let me know if I've got the naming mixed up. "Configuration" and "deployment" get thrown around interchangeably it seems, and I might have inadvertently done the same here...

Reply:
Basically I needed a way to collectively refer to the top-level properties underneath a module in `$edgeAgent`'s desired properties:

```
"modulesContent": {
  "$edgeAgent": {
    "properties.desired": {
      "modules": {
        "myModule": {      <== top-level properties of this object
          "type": "docker",
          "restartPolicy": "always",
          "status": "running",
          "settings": { ... },
          "env": { ... }
        }
      }
    }
  }
}
```